_config.yml

```diff
@@ -47,7 +47,7 @@ url:
 # (http://yourusername.github.io/repository-name)
 # and NOT your User repository (http://yourusername.github.io)
 # then add in the baseurl here, like this: "/repository-name"
-baseurl: ""
+baseurl: "https://stevenrce0.github.io/Anin_97.github.io"
```

Comment:
The baseurl should be set accordingly. Something like:

`baseurl: "https://Anin_97.github.io"`